Print cwd while user scroll between folders

Crissal1995 opened the pull request:
Compatible with both SD and NAND browsing, it shows the cwd the user is in. 
A known problem: if a path is too long, it will go on a new line (maybe autoresize or just a trimmed path with dots?).
EDIT: Fixed! Added dots ("...") to the end of cwd

Crissal1995 commented:
Fixed :eyes:

XorTroll commented:
Any screenshot about how it looks?
This could be added for v0.3

Crissal1995 commented:
@XorTroll [Here you go! ](https://imgur.com/gallery/uLpTAvD)

XorTroll commented:
Looks amazing, mind making it a bit smaller and more close to the left?
More or less like the footer below

Crissal1995 commented:
Sure, but tomorrow 😂 btw the color is ok or you prefer black?

XorTroll commented:
The color is fine, maybe I'll change it a bit, but fine for the moment.

Crissal1995 commented:
Perfect

Crissal1995 commented:
Updated with daker color, smaller font size (not 20 like footer cause it's no readable), fixed align with footer (15 px from left margin), made max characters in path displayed dipendent from the font size chosen (so if you wanna change it, there's no problem at all).

XorTroll commented:
Closing this PR as the new design will have this implemented.
